### PR TITLE
Quality of Life improvements to GTP engines

### PR DESCRIPTION
**Pull request by sethtroisi**
* Fix cc engine naming
* Change Komi to be more robust to future changes
* Move "showboard" to default python gtp engine
* Remove hardcoded 100 playout config from python gtp engine

### CODE REVIEW

File: dual_net.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 import numpy as np
 import tensorflow as tf
 from tensorflow.python.training.summary_io import SummaryWriterCache
-from tensorflow.contrib.tpu.python.tpu import bfloat16
+#from tensorflow.contrib.tpu.python.tpu import bfloat16
```
Comment (sethtroisi) [inline dual_net.py]: requires TF 1.8 while project README.md lists 1.7

Comment (amj) [reply dual_net.py]: So this isn't even used in the file?  Weird.

Comment (sethtroisi) [reply dual_net.py]: I think Brian was coping the layout of
https://github.com/tensorflow/tpu/blob/5f44b6141cea377a46c912c6b557047922ecdce9/models/official/resnet/resnet_main.py#L151

But it doesn't appear that the precision flag in this file is used

Comment (amj) [reply dual_net.py]: LGTM ok2merge

**Pull request merged**